src/main/java/org/broadinstitute/consent/http/models/Dataset.java

```diff
@@ -49,11 +49,8 @@ public class Dataset {
 
     private Boolean deletable;
 
-    private String sharingPlanDocument;
-
-    private String sharingPlanDocumentName;
-
     private FileStorageObject nihInstitutionalCertificationFile;
+    private FileStorageObject alternativeDataSharingPlanFile;
```

Comment:
I like the idea - I did consider it, but I felt that a) it would return irrelevant files, b) it puts more effort on the frontend to find the relevant file. This approach ensures that ONLY the relevant files are returned and it's easy to for the frontend to get the file that it needs - I am open to other approaches though.

Reply:
Agreed Connor, nice compromise. I also like the idea of a bucket of files, we should think more on that.